embc-app/ClientApp/src/app/provincial-admin/components/organization-maker/organization-maker.component.ts

```diff
@@ -171,7 +171,7 @@ export class OrganizationMakerComponent implements OnInit, AfterViewInit {
           // else route back to the organizations list
           if (addUsers) {
             // route to the volunteer adder page
-            this.router.navigate([`/${this.path}/volunteer`, { orgId: organization.id }]);
+            this.router.navigate([`/${this.path}/organization/${organization.id}/volunteers`]);
           } else {
```

Comment:
This code now routes to the volunteer list page for the specified organization, right?
I thought the point was to route to the volunteer maker?

Reply:
Per Curtis, this puts the user on the page where they can manage users for the org.
Another task will update the admin maker to work with the route.